core/src/main/java/io/kestra/core/models/assets/DatasetAsset.java

```diff
@@ -12,6 +12,7 @@
 
 @NoArgsConstructor
 @Plugin(aliases = DatasetAsset.ASSET_TYPE)
+@Plugin.Id()
```

Comment:
My point was, remove the alias and use `@Plugin.Id(DatasetAsset.ASSET_TYPE)` instead